<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="29339" started-at="2018-02-27T14:54:27Z" finished-at="2018-02-27T14:54:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="29339" started-at="2018-02-27T14:54:27Z" finished-at="2018-02-27T14:54:56Z">
      <class name="ua.st.selenium.LoginTest">
        <test-method status="PASS" signature="initTestSuite()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="initTestSuite" is-config="true" duration-ms="12" started-at="2018-02-27T16:54:27Z" finished-at="2018-02-27T16:54:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initTestSuite -->
        <test-method status="PASS" signature="initWebDriver()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="initWebDriver" is-config="true" duration-ms="5664" started-at="2018-02-27T16:54:27Z" finished-at="2018-02-27T16:54:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initWebDriver -->
        <test-method status="FAIL" signature="testoginNotOkInvalidEmail()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="testoginNotOkInvalidEmail" duration-ms="12105" started-at="2018-02-27T16:54:33Z" finished-at="2018-02-27T16:54:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at ua.st.selenium.LoginTest.testoginNotOkInvalidEmail(LoginTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testoginNotOkInvalidEmail -->
        <test-method status="PASS" signature="initWebDriver()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="initWebDriver" is-config="true" duration-ms="17" started-at="2018-02-27T16:54:45Z" finished-at="2018-02-27T16:54:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initWebDriver -->
        <test-method status="PASS" signature="testoginOk()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="testoginOk" duration-ms="11535" started-at="2018-02-27T16:54:45Z" finished-at="2018-02-27T16:54:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testoginOk -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:ua.st.selenium.LoginTest@685cb137]" name="tearDown" is-config="true" duration-ms="564" started-at="2018-02-27T16:54:56Z" finished-at="2018-02-27T16:54:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- ua.st.selenium.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
